packages/storybook_flutter/pubspec.yaml

```diff
@@ -13,7 +13,7 @@ dependencies:
   flutter:
     sdk: flutter
   nested: ^1.0.0
-  pointer_interceptor: ^0.9.3
+  pointer_interceptor: ^0.10.1+2
```

Comment:
Since it's not actually using any new functionality, it's better to have looser constraints, e.g. `>=0.9.0 <0.11.0`

Reply:
Updated.